hw/ip/aes/data/aes.hjson

```diff
@@ -30,13 +30,15 @@
       desc: '''
         Initial Key Registers. Loaded into the internal Full Key register upon
         starting encryption/decryption of the next block. Can only be updated
-        when the AES unit is idle. All keys registers must be updated when the
+        when the AES unit is idle. If the AES unit is busy, writes to these
```

Comment:
I might prefer "non-idle" to "busy" here - since IDLE is part of your enums and there is no formal notion of BUSY defined in the code.

Reply:
I changed it.